README.md

```diff
@@ -27,9 +28,9 @@ Most important features:
 - Multiple files can be opened at the same time like tabs in a web browser
 - The tabs can be dragged out of the window to open a new Porcupine window conveniently
 
-[Pygments]: https://pygments.org/
-[Langserver]: https://langserver.org/
-[Editorconfig]: https://editorconfig.org/
+[pygments]: https://pygments.org/
+[langserver]: https://langserver.org/
+[editorconfig]: https://editorconfig.org/
```

Comment:
Do these lowercased link definitions work with the capitalized references above?

Reply:
huh, apparently I have some plugin that automatically formats markdown files?

I assume it does, but I'm not 100% sure.

Reply:
Confirmed it works.